src/Rooms.java

```diff
@@ -0,0 +1,15 @@
+public enum Rooms {
+    A123(new RoomHour(8, 16)),
```

Comment:
A good idea to use an enum for a Room representation
One thing here -> enum/class names should be singular, not plural